sktime/transformations/series/compose.py

```diff
@@ -8,6 +8,9 @@
 from sklearn.utils.metaestimators import if_delegate_has_method
 
 from sktime.transformations.base import BaseTransformer
+from sktime.transformations.series.exponent import (
+    ExponentTransformer as _ExponentTransformer,
```

Comment:
estimators used only in the test params should be imported inside `get_test_params`